Modules/TOF/src/PostProcessDiagnosticPerCrate.cxx

```diff
@@ -54,6 +54,7 @@ void PostProcessDiagnosticPerCrate::initialize(Trigger, framework::ServiceRegist
 {
   int counter = 0;
   for (auto& i : mCrates) {
+    i.reset();
```

Comment:
why do you call `reset()` first ?

Reply:
as explained in the SkeletonPostProcessing task:
>   // We want the histogram B to be recreated at each START, so first we delete the existing one from the possible
>  // previous run, then create the new one. ROOT may crash if there are two objects with the same name at the same time.